content/docs/concepts/security/index.md

```diff
@@ -771,6 +770,51 @@ spec:
     name: "products-viewer"
 {{< /text >}}
 
+### Using Istio authorization on plain TCP protocol
+
+The above examples showed a typical way to use Istio authorization on services using HTTP protocol.
+In that case, all fields in a `ServiceRole` and `ServiceRoleBinding` are supported.
+
+Istio authorization also supports services using plain TCP protocol, like mongodb. In this case, you
+specify the `ServiceRole` and `ServiceRoleBinding` the same way as for a HTTP service, except that
+you cannot use the fields that are only applicable to HTTP services, which includes `paths` and `methods`
+in `ServiceRole`, and `group` in `ServiceRoleBinding`.
+
+For the supported constraints and properties that could be used for TCP service, please refer to
+[constraints and properties page](/docs/reference/config/authorization/constraints-and-properties/).
+
+If any HTTP only fields are used for a TCP service, the whole `ServiceRole` or `ServiceRoleBinding`
+will be ignored completely as if there is no such policy.
+
+The following example shows how to allow `bookinfo-ratings-v2` to access the mongodb service at
+port 27017:
+
+{{< text yaml >}}
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: ServiceRole
+metadata:
+  name: mongodb-viewer
+  namespace: default
+spec:
+  rules:
+  - services: ["mongodb.default.svc.cluster.local"]
+    constraints:
+    - key: "destination.port"
+      values: ["27017"]
```

Comment:
why the port list?  Does that actually work?

Reply:
That's the example port that we use in the Bookinfo-mongodb example, I updated the sentence a little bit to explain why use 27017 in the policy.